### DISCUSSION TITLE
Channel Request: Newsmax

### DISCUSSION THREAD

**Discussion by syscallfunc** [iptv-org/community#1029] [Q&A]
Not Newsmax2 but just Newsmax. It would be nice if Newsmax2 was labeled as such instead of just Newsmax.

If anyone has a link for Newsmax, that would be great.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
